M2/INSTALL

```diff
@@ -188,10 +188,11 @@ various modern operating systems:
       a tar file that can be untarred anywhere and renamed.)
     Install the programs and libraries needed for building:
 	brew update
-	brew install --build-bottle pkg-config autoconf automake gnu-tar libtool boost libmpc xz wget
-	brew install --build-bottle bdw-gc libatomic_ops gdbm glpk yasm make
+	brew install --build-bottle pkg-config autoconf automake gnu-tar libtool boost libmpc xz wget libomp
+	brew install --build-bottle bdw-gc libatomic_ops gdbm glpk yasm make cmake
```

Comment:
Do we need cmake now??

Reply:
I added that in.  The answer is no, we do not need cmake now.  We can remove it.